packages/react-positioning/src/middleware/shift.ts

```diff
@@ -0,0 +1,23 @@
+import { shift as middleware, limitShift } from '@floating-ui/dom';
```

Comment:
```suggestion
import { shift as shiftBase, limitShift } from '@floating-ui/dom';
```

```suggestion
import { shift as baseShift, limitShift } from '@floating-ui/dom';
```

nit: Just `middleware` looks confusing without looking on imports